src/backend/distributed/test/prune_shard_list.c

```diff
@@ -16,7 +16,7 @@
 
 #include <string.h>
 
-#if (PG_VERSION_NUM >= 90500 && PG_VERSION_NUM < 90600)
+#if (PG_VERSION_NUM < 90600)
```

Comment:
oddness started with the check ```PG_VERSION_NUM >= 90500 && PG_VERSION_NUM < 90600```. I do not know why we had that in the first place. removing it.